app/jobs/blog_posts_updater_job.rb

```diff
@@ -0,0 +1,7 @@
+class BlogPostsUpdaterJob < ApplicationJob
+  queue_as :default
+
+  def perform
+    Processors::BlogPostsUpdater.new.call
```

Comment:
As convention we use `BaseService` to make `Processors::BlogPostsUpdater.call` instead of using the `new`.